src/brpc/policy/http_rpc_protocol.cpp

```diff
@@ -1545,6 +1549,26 @@ void ProcessHttpRequest(InputMessageBase *msg) {
     } else {
         // A http server, just keep content as it is.
         cntl->request_attachment().swap(req_body);
+        if (imsg_guard->read_body_progressively()) {
+            accessor.set_readable_progressive_attachment(imsg_guard.get());
+            const int sc = req_header.status_code();
```

Comment:
request header has status_code?

Reply:
Sorry, I borrowed some logic from `ProcessHttpResponse` and forgot to check the details, I'll modify it. lol.